src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$cell-size: 100px;
+$default-month-length: 31;
+$days-of-week: (
+  'mon': '0',
+  'tue': '1',
+  'wed': '2',
+  'thu': '3',
+  'fri': '4',
+  'sat': '5',
+  'sun': '6',
```

Comment:
The values for the days of the week in the map `$days-of-week` are strings but should be numbers without quotes according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Use numbers to represent the index of the days.